samples/net/sockets/packet/src/packet.c

```diff
@@ -29,6 +27,8 @@ struct packet_data {
 	struct k_delayed_work send;
 };
 
+static struct packet_data packet;
```

Comment:
Please elaborate message for this commit. "Moved declaration of packet_data before definition" doesn't explain what happens here. "Move declaration of struct packet_data before definition of a variable of its type" would.

Reply:
done